src/app/modules/trainer-profile/components/editing-profile/create-aboutus/create-aboutus.component.ts

```diff
@@ -84,10 +83,14 @@ export class CreateAboutusComponent implements OnInit {
   ngOnInit(): void {
     this.getuniversities();
     this.createForm();
-
-    this.trainerService.getTrainerProfile().subscribe((profile) => {
-      this.profileForm.patchValue(profile);
-      this.prevInfo = profile;
+    this.trainerService.getTrainerProfile().subscribe((trainerProfile) => {
+      const parsedTrainer = {
+        ...trainerProfile,
```

Comment:
the educational background must be parsed to be used to display on the profile uni name and etc..